### ISSUE TITLE
No .src provided

### ISSUE THREAD

**Issue by vietj** [ceylon/ceylon-compiler#2421]
The ceylon compile module does not have .src sources in distrib

**Comment by quintesse** [comment]
In the distribution repo you mean? Actually it seems only `ceylon.language` and `ceylon-module-resolver` have them.

**Comment by vietj** [comment]
yes, but it would be good to have others too. I'm importing these jars in Maven Central and having the .src allows to produce -sources.jar that allow to debug Ceylon internals when Ceylon is consumed from a Mavenish env (for instance the ceylon-maven-plugin).
